bokehjs/src/coffee/bokeh.coffee

```diff
@@ -0,0 +1,11 @@
+Bokeh = {}
+Bokeh.version = '0.9.2'
+
+# binding the libs that bokeh uses so others can reference them
+Bokeh._                 = require 'underscore'
+Bokeh.$                 = require 'jquery'
+
+Bokeh.Backbone          = require 'backbone'
+Bokeh.Backbone.$        = Bokeh.$
+
+window.Bokeh = Bokeh
```

Comment:
I had to break this file out so that the bokeh object with our attached libraries was complete for our other modules to tap into